changelog.txt

```diff
@@ -34,6 +34,10 @@ Changed
 +++++++
 - Raises ValueError when --job-name passed by the user because that interferes with status checking (#164, #241).
 
+Fixed
++++++
+- Fix bug in FlowProject.add_operation
```

Comment:
@alacour can you make this entry a bit more descriptive about the bug, and link this pull request (just put the number in parenthesis like in the other changelog entries)?